packages/bits/src/lib/progress/progress.component.ts

```diff
@@ -40,8 +41,9 @@ export class ProgressComponent implements OnChanges {
     @Output() public cancel = new EventEmitter();
 
     public isIndeterminate = false;
+    public ariaValueNow: string | undefined;
 
-    public ngOnChanges() {
+    public ngOnChanges(changes: SimpleChanges) {
```

Comment:
Do we need `changes` here? Look like it is unnecessary.

Reply:
done